plextraktsync/trakt_api.py

```diff
@@ -389,7 +389,7 @@ def find_episode_guid(self, guid: PlexGuid, lookup: TraktLookup):
             return te.instance
         else:
             # Retry using search for specific Plex Episode
-            logger.warning(f"Retry using search for specific Plex Episode {guid.guid}")
+            logger.debug(f"Retry using search for specific Plex Episode {guid.guid}")
```

Comment:
remove this. if you want to mute the warning, use log filtering:
- https://github.com/Taxel/PlexTraktSync/pull/1102

Reply:
Why does it warn the user about this retry ? What is he supposed to do with this info ?

Reply:
I don't recall anymore why it happened, because data in trakt.tv wrong? or wrong agent?

Reply:
I'm not asking why it happen. I'm asking why should we warn user about it.
It happen when show order is different between trakt and plex. But the script handles it perfectly now, so the warning has become useless IMHO.

Reply:
but i am asking why it happens. i recall it was just temporary workaround until real problem is solved.

this fallback is slow (when doing sync) and should be dropped eventually.

Reply:
and i think it's also unsafe, this code path gets hit when no match is found, but if trakt/plex have different ordering and match is found you would be recording "watched" state to wrong episode.

i don't remember all details anymore.

Reply:
> if trakt/plex have different ordering and match is found you would be recording "watched" state to wrong episode

No, as I said, the script handles different ordering perfectly now, it cannot match a wrong episode anymore. So this warning is useless on user side, that's why i moved it to debug level.

Reply:
> this fallback is slow (when doing sync) and should be dropped eventually.

It is needed sometimes. Eg. when a show is split in many shows in trakt.

Reply:
ok. keep it as debug then